lib/main.js

```diff
@@ -6,8 +6,6 @@
 
 import {Implementation} from './Implementation.js';
 import {implementerFiles} from '../implementations/index.js';
-export {localSettings} from '../implementations/index.js';
```

Comment:
I actually think it's kind of early to remove `localSettings` as we might want to use it in the future.

Reply:
I'd rather bring it back when we need it, but if you think we are using it elsewhere (or may soon), I can bring this back now. Thoughts, @aljones15?

Reply:
it can be brought back as a minor feature so no issues if you want to keep the exports clean. in fact in general it might be better to export when needed as exports are rarely major changes to a library so good idea on your part.